client/remote.go

```diff
@@ -132,6 +132,17 @@ func (s *server) Add(r Remote) Remote {
 	return g
 }
 
+func copyTLSConfig(c *tls.Config) *tls.Config {
```

Comment:
Would it make more sense (e.g. be more robust) to copy the entire structure, in the event other fields come into play? (however unlikely)